internal/models/versionanalyzers/helm_version_analyzer.go

```diff
@@ -41,10 +41,10 @@ func (a *HelmVersionAnalyzer) ChangeNeedsVersionUpdate(changedFilesPaths []strin
 	helmRootDir = utils.GetResultOrPanic(filepath.Abs(helmRootDir))
 
 	helmTemplatesRootDir := filepath.Join(helmRootDir, "templates")
+	valuesPath := filepath.Join(helmRootDir, "values.yaml")
 	for _, path := range changedFilesPaths {
 		absPath := utils.GetResultOrPanic(filepath.Abs(path))
```

Comment:
Couldn't write any test to ensure this functionality, any idea on how to test this section?